utils/http_req.cpp

```diff
@@ -34,6 +34,7 @@ namespace
         "/system/etc/security/cacerts", // Android
         "/usr/local/share/certs",       // FreeBSD
         "/etc/pki/tls/certs",           // Fedora/RHEL
+        "/etc/ssl/certs",               // Ubuntu
```

Comment:
This really isn't needed as we do this and /cert is appended automatically:

# platform detection
if (APPLE)
    set(OPENSSL_BUILD_PLATFORM darwin64-x86_64-cc)
    set(OPENSSLDIR /usr/local/etc/openssl)
else()
    set(OPENSSL_BUILD_PLATFORM linux-generic64)
    set(OPENSSLDIR /etc/ssl)
endif()

Reply:
I guess you're not yet using our git lfs changes.